src/components/space/SyncScreen.js

```diff
@@ -223,75 +128,62 @@ class SyncScreen extends Component {
       return <SpaceNotFound />;
     }
 
-    return (
-      <>
-        <AppBar
-          position="fixed"
-          className={classNames(classes.appBar, {
-            [classes.appBarShift]: openDrawer,
-          })}
-        >
-          <Toolbar disableGutters={!openDrawer}>
-            <div
-              color="inherit"
-              className={classNames(
-                classes.menuButton,
-                openDrawer && classes.hide
-              )}
+    // if the space contains no change
+    if (isSpaceUpToDate(localSpace, remoteSpace)) {
+      return (
```

Comment:
Maybe just navigate back to the space and show a toast?

Reply:
Since the default value of localSpace and remoteSpace is an empty Map(), the first rendering will fall in this case, so I don't think I can go back from there (plus the activity is false for the first rendering).